content/client-lib-development-guide/features.textile

```diff
@@ -2141,6 +2153,10 @@ class VCDiffDecoder
 class DeltaExtras
   from: String // the id of the message the delta was generated from
   format: String //the delta format. Only vcdiff is supported as at API version 1.2
+
+class InteractionExtras:
```

Comment:
I think this should be called `Reference` (or `MessageReference` if we think `Reference` is too generic), since references aren't always going to be interactions, nor do I think the type name needs to be tied to the fact that it appears in `message.extras`

Reply:
My thinking here was to keep it consistent with `DeltaExtras`, I agree that it should have a name more related to MessageReference, though

Reply:
Ok sure, so how about `ReferenceExtras`?

Reply:
[Done](https://github.com/ably/docs/pull/1444/commits/31b19f0d93c6ec344a98e300a1e90c27cae9e572)